Tests/SmithyTestUtilTests/RequestTestUtilTests/XMLComparatorTests.swift

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import SmithyTestUtil
+import XCTest
+
+class XMLComparatorTests: XCTestCase {
```

Comment:
Tests for the XML comparator, mostly focused on the additions I made to it for XML namespaces